README.md

```diff
@@ -0,0 +1,27 @@
+# Sern CLI
+
+Our CLI is allowing you to create & streamline your Discord bot project without writing a single line of code!
+
+😁 **User Friendly Usage** <br>
+🔥 **At blazing Speeds** <br>
```

Comment:
Remove this blazing speeds thing, cli doesnt need to be blazing fast

Reply:
Ok